donate/utility/acoustic/acoustic.py

```diff
@@ -115,7 +117,7 @@ def _call_xt(self, xml):
 
     def send_mail(self, to, campaign_id, fields=None, bcc=None, save_to_db=False):
         # If we are testing, do not send emails
-        if not hasattr(settings, "IS_TESTING"):
+        if not TESTING:
```

Comment:
instead of checking for the IS_TESTING variable, check the settings whether or not the TESTING flag is True